### PR TITLE
Add wizard scripts tab Pt.1

### PR DESCRIPTION
**Pull request by glekner**
## 📝 Description

Add wizard scripts tab Pt.1
Implemented Cloudinit form section

TODO: Sysprep

## 🎥 Demo
Screen Shot 2022-03-23 at 15 56 45

### CODE REVIEW

**Comment by glekner** [comment]
@metalice please review.

File: src/views/catalog/utils/WizardVMContext/WizardVMContext.tsx [hunk 1]
```diff
@@ -29,15 +60,26 @@ export const useWizardVM = (): WizardVMContextType => {
     }
   }, [vm]);
 
+  React.useEffect(() => {
+    // whenever the wizard metadata changes, save the data in session storage
+    if (wizardMetadata) {
```
Comment (metalice) [inline src/views/catalog/utils/WizardVMContext/WizardVMContext.tsx:65]: u need this check? i can see wizardMetadata is initialized, can it be null/undfiend?

**Pull request merged**